TFKUBE-226: Add test for remote agents

nghazali opened the pull request.

jjeongatl commented:
Can you remove line 4-5 in `installer_test.go`? Go linter complains because they are unused imports

nghazali commented:
Thanks for testing. Let's clean up the code in TFKUBE-48 to avoid breaking the code. 

This maybe needed for TFKUBE-224 and TFKUBE-225 - the reason is we implemented all together, and split the relevant code to raise different PRs.

yzha645 commented on test/e2etest/installer_test.go:
```diff
@@ -27,10 +27,19 @@ func TestInstaller(t *testing.T) {
+func assertRemoteAgentList(t *testing.T, testConfig TestConfig) {
+	agentUrl := "admin/agent/configureAgents!doDefault.action"
+	url := fmt.Sprintf("https://%s@%s.%s.%s/%s", credential, product, testConfig.EnvironmentName, domain, agentUrl)
+	content := fmt.Sprintf("%s", GetPageContent(t, url))
```
Same as https://github.com/atlassian-labs/data-center-terraform/pull/103#discussion_r786342371

nghazali replied:
replied in #103

yzha645 commented on test/e2etest/installer_test.go:
```diff
@@ -27,10 +27,19 @@ func TestInstaller(t *testing.T) {
+	agentUrl := "admin/agent/configureAgents!doDefault.action"
+	url := fmt.Sprintf("https://%s@%s.%s.%s/%s", credential, product, testConfig.EnvironmentName, domain, agentUrl)
+	content := fmt.Sprintf("%s", GetPageContent(t, url))
+	assert.Contains(t,content, "There are currently 3 remote agents online.")
```
```suggestion
	assert.Contains(t, content, "There are currently 3 remote agents online.")
```
